eng/helix/content/runtests.ps1

```diff
@@ -79,6 +81,20 @@ if ([string]::IsNullOrEmpty($FeedCred)) {
     InstallDotnetSDKAndRuntime "https://dotnetclimsrc.blob.core.windows.net/dotnet" $FeedCred
 }
 
+if ($InstallPlaywright -eq "true") {
```

Comment:
Note: the external test tool dotnet-serve needs net5 to run, so that's why we need to install 5.0 as well now